### DISCUSSION TITLE
دانلود تاریخچه جزئیات معاملات

### DISCUSSION THREAD

**Discussion by rezatal** [Glyphack/pytse-client#249] [General]
باسلام و تشکر از کتابخانه مفیدی که برای استخراج دیتای بورس نوشته اید
متاسفانه با استخراج دیتا از تابع get_trade_details مشکل دارم و خطای زیر را دریافت میکنم لطفا راهنمایی بفرمایید 
asyncio.run() cannot be called from a running event loop

اگر در تابعی که دیتای میگیرد یک پارامتر بولین برای فعال و غیر فعال کردن عملکرد 
asyncio
 قرار دهید بسیار مناسب و کاربردی خواهد بود. با تشکر از شما

![image](https://github.com/Glyphack/pytse-client/assets/88963695/ce170adb-3ecb-4112-be60-6cde6b9c3e94)

![image](https://github.com/Glyphack/pytse-client/assets/88963695/44fb9aa7-2193-41aa-b31a-e825623e2c63)

**Comment by AMSeify** [comment]
اگر در محیط ژوپیتر یا پایتون IDLE استفاده می‌کنید این کد را اجرا کنید.
(نکته : قبل از اجرای دستور اجرا کنیدش)
<div , dir = "ltr">

`
import nest_asyncio; nest_asyncio.apply()
`
</div>

**Reply by rezatal** [reply]
بسیار سپاسگزارم ممنون از راهنمایی شما
